components/script/dom/domrectlist.rs

```diff
@@ -57,3 +66,18 @@ impl DOMRectListMethods for DOMRectList {
         self.Item(index)
     }
 }
+
+impl<'a> Iterator for Iter<'a> {
```

Comment:
I am maitaining `Item` signature which is `Root<DOMRect>` and `self.rects` is a vector of `JS<DOMRect>`